brainstormforce/ultimate-addons-for-gutenberg
# UAG-330 Improvement: Table of content - Added scroll to specific anchor tag feature.

### Description

- Card - https://ua-gutenberg.atlassian.net/jira/software/projects/UAG/boards/2?selectedIssue=UAG-330

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop Table of Content block on page/post.
- Visit frontend, click on heading, scroll to specific heading.
- Copy the hyperlink from url open in incognito browser and check it's scrolling to specific heading or not.

### Checklist:
